core/txpool/blobpool/evictheap_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/holiman/uint256"
 )
 
-var rand = mrand.New(mrand.NewSource(1))
+var rnd = mrand.New(mrand.NewSource(1))
```

Comment:
I renamed this because it was playing havoc with my LSP for some reason.

Reply:
then rename `mrand` to `rand` if you can